crates/sui-open-rpc/src/generate_json_rpc_spec.rs

```diff
@@ -82,6 +83,7 @@ async fn main() {
     open_rpc.add_module(EventReadApiOpenRpc::module_doc());
     open_rpc.add_module(GatewayWalletSyncApiImpl::rpc_doc_module());
```

Comment:
Lets remove all the gateway endpoints from the RPC spec, TransactionBuilderImpl and FullNodeTransactionBuilderApi is the same API.... including both in the spec will cause confusion